app/src/main/java/br/org/cesar/knot_setup_app/activity/splash/SplashActivity.java

```diff
@@ -52,4 +52,11 @@ public void apiNotConfigured(){
         Toast.makeText(this,"The backend of the webUI needs to be configured.",Toast.LENGTH_LONG).show();
     }
 
+    @Override
+    public void callbackOnConnectionError(){
+        Toast.makeText(this,"Network failure, " +
```

Comment:
Why concatenating `"Network failure, "` and `"please try again later"`?

Reply:
This is used to prevent a really long line in android (readability), I was not sure why it was done this way, but apparently, it is the default.